packages/protocol/test/signal/SignalService.t.sol

```diff
@@ -88,95 +89,4 @@ contract TestSignalService is TaikoTest {
             assertTrue(relayer.isSignalSent(Alice, signal));
         }
     }
-
```

Comment:
Tests to be added again updated or somehow not necessary anymore?

Reply:
> Tests to be added again updated or somehow not necessary anymore?

Daniel said we shall divide the SignalService tests to different files. During testing prove checks can be disabled.
I moved the L1 <> L2 (mock) test (where you helped with the encoding) to `LibTrieProof.t.sol`